docs/source/user_guide/installation/gnu-linux-mac.rst

```diff
@@ -132,7 +163,6 @@ Users can install `scikits.odes <https://github.com/bmcage/odes>`__ to utilize i
    .. code:: bash
 
       brew install openblas gcc gfortran
-      pip install wget cmake
```

Comment:
We still require `cmake` to run `pybamm_install_odes`, right?

Reply:
@santacodes Why was this resolved with no response?

Reply:
This looks to be resolved now, with https://github.com/pybamm-team/PyBaMM/pull/3914/commits/6522857446b745e01bebb358c1b1b6a0e83b7856.

Reply:
@agriyakhetarpal Actually those are the MacOS instructions, so apt is not used for that

Reply:
I added it above instead

Reply:
Oops, thanks for noticing!